### ISSUE TITLE
Dismiss button spacing is  not correct

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#2108]
### Expected behaviour
There's a space between Dismiss and the fail/success icon
### Actual behaviour
<img width="486" alt="screen shot 2017-10-04 at 11 58 52 am" src="https://user-images.githubusercontent.com/2434401/31182990-b5cdc808-a8fb-11e7-9b3e-336985b09a86.png">

### Steps to reproduce the behavior/error
Create a new user.

**Comment by rowasc** [comment]
<img width="1323" alt="7e10fca0eb31803b5193ffa91e47920b" src="https://user-images.githubusercontent.com/2434401/31183143-1bf65a46-a8fc-11e7-8c10-56b527cfbb32.png">
Same thing happens in errors.

**Comment by rowasc** [comment]
@jshorland not a blocker, but it affects different dismiss buttons across the app AFAIK

**Comment by rowasc** [comment]
<img width="679" alt="Screen Shot 2019-06-10 at 22 12 12" src="https://user-images.githubusercontent.com/2434401/59236498-d682db00-8bcc-11e9-944f-38be949ad33f.png">
This looks like it was fixed . Tried it in several mobile screen resolutions without problems , and also larger screens seem fine. Closing
